Source/Core/Plugins/EventProcessor/Default/03_ManualStackingPlugin.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.Threading.Tasks;
+using Exceptionless.Core.Pipeline;
+using Exceptionless.Core.Extensions;
+using Exceptionless.Core.Models;
+
+namespace Exceptionless.Core.Plugins.EventProcessor {
+    [Priority(3)]
+    public class ManualStackingPlugin : EventProcessorPluginBase {
+        public override Task EventProcessingAsync(EventContext context) {
+            var data = context.Event.Data;
+            if (!data.ContainsKey(Event.KnownDataKeys.ManualStackingKey)
+                || data[Event.KnownDataKeys.ManualStackingKey] == null)
+                return Task.CompletedTask;
+
+            context.StackSignatureData.AddItemIfNotEmpty(nameof(Event.KnownDataKeys.ManualStackingKey), data[Event.KnownDataKeys.ManualStackingKey].ToString());
```

Comment:
This is what will be displayed as the Stacking Information. So, yes, it would resolve to "ManualStackingKey". I guess what would you want to see there:
![image](https://cloud.githubusercontent.com/assets/6343727/12965485/892034be-d024-11e5-99a3-397c614a7746.png)